### PR TITLE
Change stylesheets to Sass 

### PR DESCRIPTION
**Pull request by sdegetaus**
#### Changes
- Changed the directory of admin.css
- Added node-sass and sass-loader
- Added webpack watch options

Although the stylesheets are still small and manageable, I think that using sass from now on will save effort later.
If you favor these changes, then I can change the other style files to .sass 🙂(plus adding variables and such).

### CODE REVIEW

**Comment by dannyvankooten** [comment]
Hey @sdegetaus,

Agreed - was meaning to add SASS eventually as the selector prefix was already getting a bit annoying, so thank you for helping out here!

I'm going to add a few comments on things I'm not sure of.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "watch": "NODE_ENV=development webpack --watch",
+    "watch": "set NODE_ENV=development && webpack --watch --progress",
```
Comment (dannyvankooten) [inline package.json]: Is this intentional? In my tests, the `set` command does not work properly on Unix and does not actually set the environment variable properly. What's wrong with `NODE_ENV=development webpack --watch --progress`?

File: assets/src/js/admin.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import './admin.css';
+import '././sass/admin.scss';
```
Comment (dannyvankooten) [inline assets/src/js/admin.js]: Can you change this to `import './sass/admin.scss';`?

File: package.json (hunk 1 above)
Comment (sdegetaus) [reply package.json]: I was getting the error `'NODE_ENV' is not recognized as an internal or external command`. After a quick research, I found [this StackOverflow question](https://stackoverflow.com/questions/11928013/node-env-is-not-recognized-as-an-internal-or-external-command-operable-comman).

It seems that on Windows you need to write `set` and `&&` for it to work properly 😕.

However, I think there are some packages that solve this cross-platform inconvenience, I will take a look at that 🙂, for the time being, I don't mind discarding these lines whenever I commit something.

File: assets/src/js/admin.js (hunk 2 above)
Comment (sdegetaus) [reply assets/src/js/admin.js]: Ah! That was my mistake. But just to be clear:
Do you want the new Sass files under `src` or under `js`? I was thinking that it should be `koko-analytics > assets > src > sass`

File: package.json (hunk 1 above)
Comment (sdegetaus) [reply package.json]: Adding this module to my environment fixed the issue:
`npm install -g win-node-env`

File: assets/src/js/admin.js (hunk 2 above)
Comment (dannyvankooten) [reply assets/src/js/admin.js]: `assets > src > sass` works for me!

File: assets/src/js/components/settings.js [hunk 3]
```diff
@@ -42,10 +42,10 @@ function Component() {
                     <div className={"grid"} style={"margin-bottom: 24px;"}>
                         <div style={"grid-column: span 4;"}>
                             <h1>{i18n['Settings']}</h1>
-                            <form method={"POST"} onsubmit={onsubmit}>
+                            <form method={"POST"} onSubmit={onSubmit}>
```
Comment (dannyvankooten) [inline assets/src/js/components/settings.js]: Looks like your editor is changing this automatically, but unfortunately `onSubmit` & `onChange` breaks the application as Mithril.js uses lowercased event attributes.

File: koko-analytics.php [hunk 4]
```diff
@@ -1,16 +1,14 @@
...
+Plugin URI: https://dvk.co/#utm_source=wp-plugin&utm_medium=koko-analytics&utm_campaign=plugins-page
 Description: Privacy-friendly analytics for your WordPress site.
 Author: ibericode
 Author URI: https://ibericode.com/#utm_source=wp-plugin&utm_medium=koko-analytics&utm_campaign=plugins-page
-Author Email: support@kokoanalytics.com
 Text Domain: koko-analytics
 Domain Path: /languages/
-License: GPL-3.0
-License URI: https://www.gnu.org/licenses/gpl-3.0.txt
+License: GPL v3
```
Comment (dannyvankooten) [inline koko-analytics.php]: Looks like some (newer) commits were accidentally reverted, can you change the PR to include the latest commit (or at least not revert it)?

Comment (sdegetaus) [reply koko-analytics.php]: Agh, so sorry. I am quite new on doing pull requests from a fork, I will fix both problems.

Comment (sdegetaus) [reply koko-analytics.php]: @dannyvankooten I applied your recent changes to my newest commits. Sorry about that!

**Comment by dannyvankooten** [comment]
:tada: Looks good to me @sdegetaus, merged. Thanks a ton!

**Comment by sdegetaus** [comment]
@dannyvankooten Thank you! Hopefully it won't be my last contribution!

**Pull request merged**